tuf/api/metadata.py

```diff
@@ -702,6 +702,21 @@ def _verify_length(
                 f"expected length {expected_length}"
             )
 
+    @staticmethod
+    def _validate_hashes(hashes: Dict[str, str]) -> None:
+        if not isinstance(hashes, dict):
+            raise TypeError(f"Hashes must be a dictionary, got {type(hashes)}")
```

Comment:
This check could now be dropped -- the error we get from calling `hashes.items()` on something that isn't a dict is not as good but it does raise an exception without extra code...

Reply:
I dropped it and force pushed, we are raising build-in errors anyway.